providers/emailjs/CHANGELOG.md

```diff
@@ -3,41 +3,41 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [0.2.5](https://github.com/novuhq/notifire/compare/v0.2.4...v0.2.5) (2021-11-05)
+## [0.2.5](https://github.com/novuhq/novu/compare/v0.2.4...v0.2.5) (2021-11-05)
 
-**Note:** Version bump only for package @notifire/emailjs
+**Note:** Version bump only for package @notifire/emailjs (now @novu/emailjs)
```

Comment:
I think it's helpful to clarify the current package names. As this was all quite confusing.